.github/workflows/openapi-gen.yml

```diff
@@ -0,0 +1,99 @@
+name: OPENAPI-GEN
+
+on:
+  workflow_call:
+    inputs:
+      docs_path:
+        type: string
+        required: true
+      owner:
+        type: string
+        required: true
+      pr_repository:
+        type: string
+        required: true
+      pr_repository_path:
+        type: string
+        required: true
+      pr_repository_base_branch:
+        type: string
+        required: true
+
+    secrets:
+      PAT:
+        required: true
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: this
+
+      - name: Get Generator
+        run: |
+          wget https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/7.0.1/openapi-generator-cli-7.0.1.jar -O openapi-generator-cli.jar
```

Comment:
I don't _think_ doing `wget`s at runtime is really an issue, I did want to avoid it for potential security issues. I don't necessarily think it's really a problem though, let me know if you think it could be.

Reply:
If it were for `../latest.jar` then it'd be more of an issue rather than a specific version, but yeah, I share your trepidation